db/migrate/20151021014315_create_course_assessment_question_text_response.rb

```diff
@@ -0,0 +1,26 @@
+class CreateCourseAssessmentQuestionTextResponse < ActiveRecord::Migration
+  def change
+    create_table :course_assessment_question_text_responses do |t|
```

Comment:
Unused block argument - `t`. You can omit the argument if you don't care about it.